SCons/Tool/MSCommon/vs.py

```diff
@@ -436,20 +452,29 @@ def get_installed_visual_studios(env=None):
             if vs.get_executable(env):
                 debug('found VS %s', vs.version)
                 InstalledVSList.append(vs)
+                if vs.is_express and vs.verstr not in InstalledVSMap:
+                    if _VSEXPRESS_USE_VERSTR:
+                        InstalledVSMap[vs.verstr] = vs
                 InstalledVSMap[vs.version] = vs
     return InstalledVSList
 
+def _get_installed_vss(env=None):
+    get_installed_visual_studios(env)
+    versions = list(InstalledVSMap.keys())
+    return versions
+
 def reset_installed_visual_studios() -> None:
     global InstalledVSList
     global InstalledVSMap
+    debug('')
```

Comment:
?

Reply:
These two calls are effectively equivalent in the output log: `debug('')` and `debug('called')`.   
   
In this case, the string payload is not important but rather the *code location* (i.e., source file name, function name, and line number) in the *log prefix* which is effectively the entry point of the reset function.
   
Including the entry point for the reset function provides more context reset call chain and log records.  The string payload itself is not adding any informational context.